content/en/blog/_posts/2024-10-28-memory-manager-moves-to-ga/index.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: blog
+title: "Memory Manager goes GA"
+date: 2024-10-28
+slug: memory-manager-goes-ga
+author: >
+  [Talor Itzhak](https://github.com/Tal-or) (Red Hat)
+---
+
+With Kubernetes 1.32, the Memory Manager has officially graduated to General Availability (GA),
+marking a significant milestone in the journey toward efficient and predictable memory allocation for containerized applications.
+Since Kubernetes v1.22, where it graduated to [Beta](https://kubernetes.io/blog/2021/08/11/kubernetes-1-22-feature-memory-manager-moves-to-beta/)
+the Memory Manager proved itself reliable, stable and a good complementary feature for the [CPU Manager](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/).
+The Memory Manager enables the user to allocate exclusive
+memory for `Guaranteed` [QoS class](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes) pods.
+Most of the changes introduced since the Beta are bug fixes, internal refactoring and 
+observability improvements, such as metrics and better logging.
+
+## New metrics for Memory Manger
+
+As part of the effort
+to increase the observability of Memory Manager, new metrics have been added
+to provide some statistics on memory allocation patterns.
+
+
+**memory_manager_pinning_requests_total**
+tracks the number of times the pod spec required the memory manager to pin memory pages.
+
+**memory_manager_pinning_errors_total**
+tracks the number of times the pod spec required the memory manager 
+to pin memory pages, but the allocation failed.
```

Comment:
We can call this "observability improvements", hint the fact there's an ongoing effort to add metrics and slowly make the kubelet more observable, and then focus on what memory manager provides

Reply:
Cool, thanks!